frameworks/Rust/may-minihttp/Cargo.toml

```diff
@@ -17,15 +17,18 @@ nanorand = { version = "0.7", default-features = false, features = ["std", "wyra
 buf-min = { version = "0.7", features = ["bytes"] }
 yarte = { version = "0.15", features = ["bytes-buf", "json"] }
 
-may = { version = "0.3.25", default-features = false }
+may = "0.3"
 may_postgres = { git = "https://github.com/Xudong-Huang/may_postgres.git" }
 may_minihttp = { git = "https://github.com/Xudong-Huang/may_minihttp.git" }
 
 [profile.release]
 opt-level = 3
 codegen-units = 1
 panic = 'abort'
-lto = "thin"
+lto = 'thin'
 debug = false
 incremental = false
 overflow-checks = false
+
+[patch.crates-io]
+may = { git = "https://github.com/Xudong-Huang/may.git", branch = "no_default_features" }
```

Comment:
Ditto.

Reply:
already remove this, please see the latest code.